pycbc/strain.py

```diff
@@ -354,8 +381,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
                              pd_taper_window) )
         gate_data(strain, gate_params)
 
-
+    strain.injcutter = injcutter
```

Comment:
@spxiwh I'm not a fan of the name ``injcutter`` as it's jargon and not very specific. ``injcutter`` could be cutting the injections due to SNR, or intrinsic parameter, or... Can you come up with a better name that's more obvious to the novice user?